src/main/java/org/folio/circulation/domain/notice/session/PatronActionSessionService.java

```diff
@@ -107,9 +113,12 @@ public PatronActionSessionService(
     MultipleRecords<PatronSessionRecord> records) {
 
     if (records == null) {
+      log.info("Records are empty, nothing to delete.");
```

Comment:
Just by looking at this phrase in the log, it's not that easy to understand what exactly "Records" is. Should we maybe add so me more details?